tests/test_metrics_service.py

```diff
@@ -53,6 +61,7 @@ class MetricsServiceTestCase(unittest.TestCase):
     )
     def test_log(self, mock_execute_ingest, mock_MONGO_CONNECTION, mock_post_event):
         mock_MONGO_CONNECTION._client = MetricsServiceTestCase.client_mock
+        # Test when user uuid is not provided
```

Comment:
More consistent with Rails:

```suggestion
        # Test when metrics uuid is not provided
```